frontend/src/components/abfragevarianten/SachbearbeitungComponent.vue

```diff
@@ -37,6 +37,41 @@
           />
         </v-col>
       </v-row>
+      <v-row>
+        <v-col
+          cols="12"
+          md="6"
+        >
+          <v-checkbox
+            id="sobon_berechnung"
+            ref="sobonBerechnungCheckbox"
+            v-model="abfragevarianteSachbearbeitung.isASobonBerechnung"
```

Comment:
Soll die sobonursächliche Berechnung durchgeführt werden wenn dieses Attribut auf false ist? 

Falls diese nicht durchgeführt werden soll dies implementiert oder zumindest in einer eigenen Userstory festgehalten werden.